docs/resources.md

```diff
@@ -45,3 +45,6 @@ object HttpSuite extends IOSuite {
 }
 ```
 
+```scala mdoc:passthrough
+println(weaver.docs.Output.runSuites(HttpSuite).unsafeRunSync())
```

Comment:
One worry is that this test suite actually runs an HTTP request to httpbin...

I'll probably revert this, don't want it to break during CI.

Reply:
Changed in 3fd0aee, arguably better example as a whole: http://weaver-site.surge.sh/weaver-test/docs/resources